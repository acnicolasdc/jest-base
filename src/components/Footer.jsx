import React from 'react';
import '../styles/components/Footer.styl';

const Footer = () => (
  <div className="Footer">
    <p className="Footer-title">Jest Testing Store</p>
    <p className="Footer-copy">Platzi course</p>
  </div>
);

export default Footer;
