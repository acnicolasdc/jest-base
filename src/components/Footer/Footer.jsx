import React from 'react';
import './Footer.styl';

const Footer = () => (
  <div className="Footer">
    <p className="Footer-title">Jest Testing Store</p>
    <p className="Footer-copy">2020/07/12</p>
  </div>
);

export default Footer;
