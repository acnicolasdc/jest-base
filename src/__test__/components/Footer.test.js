import React from 'react';
import { mount } from 'enzyme';
import Footer from '../../components/Footer/Footer';

describe('<Footer />', () => {
  const footer = mount(<Footer />);
  test('Component Footer render ', () => {
    expect(footer.length).toEqual(1);
  });
  test('Component Footer title ', () => {
    expect(footer.find('.Footer-title').text()).toEqual('Jest Testing Store');
  });
});
