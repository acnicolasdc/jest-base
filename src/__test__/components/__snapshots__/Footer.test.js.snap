// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot Testing the Footers component UI 1`] = `
<div
  className="Footer"
>
  <p
    className="Footer-title"
  >
    Jest Testing Store
  </p>
  <p
    className="Footer-copy"
  >
    2020/07/12
  </p>
</div>
`;
